sao.js

```diff
@@ -107,6 +107,17 @@ module.exports = {
           ? true
           : 'Please include a valid GitHub.com URL without a trailing slash';
       }
+    },
+    trimWhitespaceMarkdown: {
+      message: 'Trim trailing whitespace in markdown files?',
```

Comment:
You don't need to append `?` in the string, it is prefixed.

Btw if you want to test your fork, you can do so locally by running `./cli.js some-folder-name`.

Reply:
I was testing with `npm link`. I could have sworn I saw some lines that had question marks. I'll remove it.